client/src/app/core/definitions/key-types.ts

```diff
@@ -4,3 +4,4 @@ export type Collectionfield = string;
 export type Collection = string;
 export type Field = string;
 export type Id = number;
+export type HTML = string;
```

Comment:
Please call this `UnsafeHtml`. Angular uses HTML validators to actually display it in the dom. We should explicit distinguish and be not confused with angulars' parsed and trusted HTML.